distributed/scheduler.py

```diff
@@ -1345,10 +1351,13 @@ def __init__(
         self.total_nthreads = 0
         self.total_occupancy = 0.0
         self.unknown_durations: dict[str, set[TaskState]] = {}
+        self.last_root_worker: WorkerState | None = None
+        self.last_root_worker_tasks_left: int = 0
+        self.queued = queued
         self.unrunnable = unrunnable
         self.validate = validate
         self.workers = workers
-        self.running = {
+        self.running: set[WorkerState] = {
```

Comment:
Pretty sure this is redundant.
Eventually I want to clean up SchedulerState and Scheduler and change all attribute declarations to C++-style, like it already happens in the other classes of this module as well as in worker_state_machine.py.

Reply:
Without this, `reveal_type(self.running)` after gives `note: Revealed type is "builtins.set[Any]"`. So it's not necessary, but I added it for improved type coverage and editor auto-complete.